File: Homework 2/JAVA/AlphaString.java
//set string makes a proper string using single name
//get returns fixed string

public class AlphaString {
	private String s;

	public AlphaString(String s){
		this.s = s;
	}
	public AlphaString(){}

	public String get() {
		return s;
	}

	public void set(String s) {
		this.s = s;

	}
}

File: Homework 2/C++_assign1/AlphaString.cpp
#include <string>

namespace std;

class AlphaString
{
private:
string s;

public:
	AlphaString::AlphaString(string s)
	{
		this->s = s;
	}
	AlphaString::AlphaString()
	{
	}

	String AlphaString::get(void)
	{
		return s;
	}

	String AlphaString::set(string s)
	{
		this->s = s;
		return this->s;

	}
};
